src/index.js

```diff
@@ -1,35 +1,382 @@
+  // });
+
+  document
+    .getElementById('todo-list-element')
+    .appendChild(todoListItemContainer);
+
+  document
+    .getElementById('todo-list-element')
+    .appendChild(document.createElement('hr'));
+
+  console.log(todoListElement.id);
+
+  todoListCheck.addEventListener('click', () => {
+    if (todoListCheck.checked === true) {
+      todoListCheck.nextElementSibling.style.textDecoration = 'line-through';
+      todoListCheck.nextElementSibling.style.opacity = '0.5';
+
+      todoArray[indx].completed = true;
+    } else {
+      todoListCheck.nextElementSibling.style.textDecoration = 'none';
+      todoListCheck.nextElementSibling.style.opacity = '1';
+
+      todoArray[indx].completed = false;
+    }
+    localStorage.setItem('todoList', JSON.stringify(todoArray));
+  });
+
+  const bin = document.createElement('span');
+  bin.classList.add('hover-bin');
+  todoListElement.parentElement.appendChild(bin);
+
+  todoListElement.addEventListener('click', () => {
+    console.log(todoListElement.getAttribute('data-clicked'));
+
+    if (!todoListElement.getAttribute('data-clicked')) {
+      todoListElement.setAttribute('data-clicked', 'true');
+
+      todoListElement.contentEditable = true;
+      todoListElement.classList.remove('ellipsis');
+      todoListElement.focus();
+      todoListElement.parentElement.style.backgroundColor = '#f6d788';
+      todoListElement.style.cursor = 'text';
+      todoListElement.nextElementSibling.style.display = 'none';
+      bin.innerHTML = '&#128465;';
+      bin.style.display = 'inline';
+      bin.onclick = () => {
+        document.querySelector(`#todo-list-container-${indx} + hr`).remove();
+        document.getElementById(`todo-list-container-${indx}`).remove();
+
+        todoArray.splice(indx, 1);
+        todoArray.forEach((todo, ind) => {
+          todo.index = ind;
+        });
+        localStorage.setItem('todoList', JSON.stringify(todoArray));
+      };
+    } else {
+      todoListElement.removeAttribute('data-clicked');
+
+      todoListElement.contentEditable = false;
+      todoListElement.classList.add('ellipsis');
+      todoListElement.parentElement.style.backgroundColor = 'white';
+      todoListElement.style.cursor = 'pointer';
+      todoListElement.nextElementSibling.style.display = 'inline';
+      bin.innerHTML = '';
+      bin.style.display = 'none';
+
+      const indexOfTodoElement = indx;
+
+      console.log(indexOfTodoElement);
+
+      todoArray.forEach((todo) => {
+        if (todo.index === Number(indexOfTodoElement)) {
+          todo.description = todoListElement.innerText;
+        }
+      });
+      localStorage.setItem('todoList', JSON.stringify(todoArray));
+    }
+  });
+};
+
+document.getElementById('todo-enter').addEventListener('click', () => {
+  const descriptionTodo = InputElementTodo.value;
+  InputElementTodo.value = '';
+  localStorage.setItem('input', InputElementTodo.value);
+  todoArray.push({
+    description: descriptionTodo,
```

Comment:
- [ ] As the previous reviewer suggested, kindly ensure that you have created a new JavaScript file for the new functionality you added for this milestone. For example, you can create a file for `adding`, `removing`, and `editing` a task then import them into your `index.js` main file. :+1: 

> Example:
```
 src > modules >
                 delete.js
                 edit.js
                 add.js   etc.
```

Reply:
completed.